### PR TITLE
RavenDB-17391 Redesign preview icon, add preview-off icon

### PR DESCRIPTION
**Pull request by mateuszbartosik**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-17391/Create-new-icon-for-close-preview

_Please delete below the options that are not relevant_

### Type of change
- Optimization

### How risky is the change?
- Not relevant

### Backward compatibility
- Not relevant

### Is it platform specific issue?
- No

### Documentation update
- No documentation update is needed 

### Testing 
- It has been verified by manual testing

### Is there any existing behavior change of other features due to this change?
- No

### UI work
- No UI work is needed

### CODE REVIEW

File: src/Raven.Studio/wwwroot/Content/css/styles-common.less [hunk 1]
```diff
@@ -431,8 +431,7 @@ i {
 .btn {
     [class*=" icon-"],
     [class^="icon-"] {
-        top: .1em;
-        transform: scale(1.1);
+        top: .2em;
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/Content/css/styles-common.less:434]: this is big change. Did you check few other icons? Does it look good?

File: src/Raven.Studio/wwwroot/Content/css/bootstrap/type.less [hunk 2]
```diff
@@ -2,6 +2,10 @@
 // Typography
 // --------------------------------------------------
 
+.text-white {
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/Content/css/bootstrap/type.less]: @kwiato do you think this is good location for this change? Does it look good in different themes? Usually we didn't have separate colors for icons but all shared same color.

Comment (ml054) [reply src/Raven.Studio/wwwroot/Content/css/bootstrap/type.less]: so I'm not sure we even need this class.

Comment (kwiato) [reply src/Raven.Studio/wwwroot/Content/css/bootstrap/type.less]: in cloud we have a ` .text-emphasis` class

Comment (mateuszbartosik) [reply src/Raven.Studio/wwwroot/Content/css/bootstrap/type.less]: The class has been removed

File: src/Raven.Studio/wwwroot/Content/css/styles-common.less (hunk 1 above)
Comment (mateuszbartosik) [reply src/Raven.Studio/wwwroot/Content/css/styles-common.less:434]: Yes, I've gone through most of the pages and the buttons looked fine. In fact, the icons look more center-aligned now. For reference, I've uploaded the comparison (based on Notification Center) here: https://issues.hibernatingrhinos.com/issue/RavenDB-17391/Create-new-icon-for-close-preview#focus=Comments-67-385789.0-0

Comment (ml054) [reply src/Raven.Studio/wwwroot/Content/css/styles-common.less:434]: great thanks

**Review by ml054 — APPROVED** [review]

**Pull request merged**